cl/users/templates/includes/webhooks_htmx/webhooks-test-webhook.html

```diff
@@ -17,7 +17,7 @@
         </div>
       </div>
     </div>
-  {% if da_type == webhook.event_type %}
+  {% if  webhook.event_type in da_type %}
```

Comment:
Seems like we need some variable cleanup here too, right?

Reply:
Yeah, updated to `event_types`.